Web/coolprop/LowLevelAPI.rst

```diff
@@ -25,6 +25,46 @@ which yields the output:
 
 .. literalinclude:: snippets/AbstractState1.cxx.output
 
+Here is an example of the shared library usage with Julia wrapper::
+
+    julia> import CoolProp
+
+    julia> PT_INPUTS = CoolProp.get_input_pair_index("PT_INPUTS")
```

Comment:
Exactly, I think it would be great to link it to the low level documentation.
(I do not know yet how internal links in the documentation works)